azure-kusto-data/azure/kusto/data/client.py

```diff
@@ -72,6 +74,17 @@ def __init__(self, kcsb: Union[KustoConnectionStringBuilder, str]):
         self._session.mount("http://", adapter)
         self._session.mount("https://", adapter)
 
+    def close(self):
+        if not self._is_closed:
+            self._session.close()
+        super().close()
+
+    def __enter__(self):
```

Comment:
Maybe move it to the base?

Reply:
`__enter__` is used for "with",` __aenter__` is used for async with 
Since it's wrong to initialize an async client with a normal "with", the method is not available